src/chain/store/chain_store.rs

```diff
@@ -10,6 +10,7 @@ use crate::interpreter::VMTrace;
 use crate::libp2p_bitswap::{BitswapStoreRead, BitswapStoreReadWrite};
 use crate::message::{ChainMessage, Message as MessageTrait, SignedMessage};
 use crate::networks::ChainConfig;
+use crate::rpc::eth::Hash as EthHash;
```

Comment:
non-blocking: I would `use crate::rpc::eth` and refer to `eth::Hash` rather than renaming, because it's more amenable to (current) automated refactoring

Reply:
Sure.